packages/synapse-react-client/src/components/ColumnFilter.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import { useQueryContext } from './QueryContext/QueryContext'
+import Select, {
+  components,
+  ControlProps,
+  GroupBase,
+  MultiValue,
+} from 'react-select'
+import {
+  ColumnMultiValueFunction,
+  ColumnMultiValueFunctionQueryFilter,
+  FacetColumnResultValues,
+  QueryFilter,
+} from '@sage-bionetworks/synapse-types'
+import { useQuery } from '@tanstack/react-query'
+
+export type FilterProps = {
+  filterColumnName?: string
+}
+
+export const Control = ({
+  children,
+  ...rest
+}: ControlProps<any, boolean, GroupBase<any>>) => {
+  return (
+    <components.Control {...rest} className="form-control">
+      {children}
+    </components.Control>
+  )
+}
+
+const ColumnFilter: React.FC<FilterProps> = props => {
```

Comment:
Prefer using `function` and removing `React.FC` (which I think is deprecated)
```suggestion
function ColumnFilter (props: FilterProps) {
```

Reply:
Could you add some tests for this component? The QuerySortSelector has a corresponding QuerySortSelector.test.tsx file that you can use as a base. Unfortunately, that one is older, so it's not type-checked and co-located with the QuerySortSelector component file.

For the `ColumnFilter.test.tsx`, could you co-locate the file with your new component file, and ensure it passes typechecking?